storm.yaml

```diff
@@ -4,27 +4,36 @@
 mesos.master.url: "zk://localhost:2181/mesos"
 storm.zookeeper.servers:
     - "localhost"
-nimbus.host: "localhost"
 # -----------------------------------------------------------
 
 
-### If this is not false then you need to add about 128m of mem usage to executor to account for the logviewer
-#supervisor.autostart.logviewer: false
-topology.mesos.executor.mem.mb: 1128  ## default is 1000
+# Worker resources
+topology.mesos.worker.cpu: 1.0
+### Worker heap with 20% overhead
+topology.mesos.executor.mem.mb: 1250
+worker.childopts: "-Xmx1000m"
 
-# The default behavior is to launch the logvierwer unless autostart is false
-logviewer.port: 8000
+# Supervisor resources
+topology.mesos.executor.cpu: 0.1
+topology.mesos.executor.mem.mb: 500 # Supervisor memory, with 20% overhead
+supervisor.childopts: "-Xmx256m"
+
+# The default behavior is to launch the logviewer unless autostart is false.
+# If you enable the logviewer, you'll need to add memory overhead to the
+# executor for the logviewer.
 logviewer.childopts: "-Xmx128m"
 logviewer.cleanup.age.mins: 10080
 logviewer.appender.name: "A1"
-
-
-# You should not need to change anything below this line
-#--------------------------------------------------------
+supervisor.autostart.logviewer: true
```

Comment:
Yeah, the storm clojure is especially hard due to the large number of storm-specific macros and the un-functional nature of it (tons of state usage and inter-dependencies and long functions).

To be clear, I don't consider this to be supervisor-specific so much as Mesos Executor-specific;  i.e., the executor containers have their separate sandboxes for the log files, and those need to be exposed separately.  Unless you do as my company does and send your logs out of the container into the native host's filesystem.  Then you can have a single logviewer.

Can you please clarify: "This should let you run multiple supervisors, but not from the same Storm cluster."?   We will *definitely* still be running multiple topologies on each host from a single storm cluster overlay.  And I honestly don't care about the logviewer...